packages/arb-validator/validator/validator.go

```diff
@@ -345,7 +350,20 @@ func (validator *Validator) Run(ctx context.Context, recvChan <-chan ethbridge.N
 				validator.latestHeader = newHeader
 				err := validator.timeUpdate()
 				if err != nil {
-					log.Println("Error processing time update", err)
+					//log.Printf("Validator %v: Error processing time update - %v\n", validator.Name, err)
+					if errstat, ok := err.(*bridge.Error); ok {
```

Comment:
If this is the only type of error that can be returned from `timeUpdate` it'd probably be better to change the signature and avoid this cast